tests/functional/conftest.py

```diff
@@ -17,68 +17,37 @@ def fixture_dir(test_dir):
     return test_dir / "functional" / "fixtures"
 
 
-def reset_gitlab(gl):
-    # previously tools/reset_gitlab.py
+def reset_gitlab(gl: gitlab.Gitlab) -> None:
+    """Delete resources (such as projects, groups, users) that shouldn't
+    exist."""
     for project in gl.projects.list():
-        logging.info(f"Marking for deletion project: {project.path_with_namespace!r}")
         for deploy_token in project.deploytokens.list():
             logging.info(
-                f"Marking for deletion token: {deploy_token.username!r} in "
+                f"Deleting deploy token: {deploy_token.username!r} in "
                 f"project: {project.path_with_namespace!r}"
             )
```

Comment:
I think we're now logging with info both in and outside the function - could bring it together at some point.